src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -125,4 +125,8 @@ public void linkOnCommentsLink() {
         AppPage.getNewPageInstance(browser, StudentCommentsPage.class);
         
     }
+    
+    public void waitForCellHoverToDisappear() {
+        waitForElementToDisappear(By.className("cell-hover"));
```

Comment:
Rather than removing the class, I think we could try adding code to hover over a specific cell with Selenium. We should test that cell-hover class appears when the cells are moused over, unless it's too hard to do so?